executioncontext/workspacecontext/src/main/java/org/jetbrains/bsp/bazel/workspacecontext/TargetsSpec.kt

```diff
@@ -23,24 +25,62 @@ internal object TargetsSpecMapper : ProjectViewToExecutionContextEntityMapper<Ta
     private const val NAME = "targets"
 
     override fun map(projectView: ProjectView): Try<TargetsSpec> =
+            if (deriveTargetsFlags(projectView.deriveTargetsFlag!!))
```

Comment:
for null (when the flag is not specified it will fail)

Reply:
Fixed